README.md

```diff
@@ -45,160 +88,88 @@ To run a specific unittest:
 
     source ~/certbot/tests/integration/_common.sh
     certbot_test certonly -a standalone -d example.com
 
-Your local Boulder instance uses a fake DNS server that returns 127.0.0.1 for
-any query, so you can use any value for the -d flag. You can also override that
-value by setting the environment variable FAKE_DNS=1.2.3.4
+Your local Boulder instance uses a fake DNS resolver that returns 127.0.0.1 for any query, so you can use any value for the -d flag. If you want to use another DNS resolver you can by setting the environment variable FAKE_DNS=1.2.3.4.
 
-By default `certbot_test` will use `-server 127.0.0.1:4000/directory`, the ACME v1 API endpoint. To test ACME v2 (e.g. for wildcard issuance) please use `--server 127.0.0.1:4001/directory` to access the ACME v2 endpoint on port `4001`.
+By default Certbot will connect to the ACME v2 API over HTTP. You can customize the `SERVER` environment variable with an alternative ACME directory URL if required.
 
-Component Model
----------------
+### Working with another ACME Client
 
-The CA is divided into the following main components:
+Once you have followed the Boulder development environment instructions and have
+started the containers you will find the ACME endpoints exposed to your host at
+the following URLs:
 
-1. Web Front End
-2. Registration Authority
-3. Validation Authority
-4. Certificate Authority
-5. Storage Authority
-6. OCSP Updater
-7. OCSP Responder
+* ACME v1, HTTP: `http://localhost:4000/directory`
+* ACME v2, HTTP: `http://localhost:4001/directory`
+* ACME v1, HTTPS: `https://localhost:4430/directory`
+* ACME v2, HTTPS: `https://localhost:4431/directory`
 
-This component model lets us separate the function of the CA by security context.  The Web Front End and Validation Authority need access to the Internet, which puts them at greater risk of compromise.  The Registration Authority can live without Internet connectivity, but still needs to talk to the Web Front End and Validation Authority.  The Certificate Authority need only receive instructions from the Registration Authority. All components talk to the SA for storage, so lines indicating SA RPCs are not shown here.
+To access the HTTPS versions of the endpoints you will need to configure your ACME client software to use a CA truststore that contains the `test/wfe-tls/minica.pem` CA certificate. See [the `test/wfe-tls` README](https://github.com/letsencrypt/boulder/master/test/wfe-tls/README) for more information.
 
-```
+Your local Boulder instance uses a fake DNS resolver that returns 127.0.0.1 for any query, allowing you to issue certificates for any domain as if it resolved to your localhost. If you want to use another DNS resolver you can by setting the environment variable FAKE_DNS=1.2.3.4.
```

Comment:
Same comment as above: FAKE_DNS doesn't change which DNS resolver you use, just the answers you get.